esmvalcore/config/_config_object.py

```diff
@@ -431,6 +434,39 @@ def start_session(self, name: str) -> Session:
             session = Session(config=self.copy(), name=name)
         return session
 
+    def update_from_dirs(self, dirs: Iterable[str | Path]) -> None:
+        """Update configuration object from directories.
+
+        This will first search for all YAML files within the given directories
+        and merge them together using :func:`dask.config.collect` (values in
+        the latter directories are preferred to those in the former). Then, the
```

Comment:
```suggestion
        and merge them together using :func:`dask.config.collect` (if identical values
        are provided in multiple files, the value from the last file will be used).
        Then, the
```
Maybe rephrase like this? As far as I understand it, 'the latter' and 'the former' are typically used when there are only two options mentioned.